tests/test_spec.py

```diff
@@ -0,0 +1,32 @@
+import json
+from pathlib import Path
+
+import mdformat
+from mdformat._util import is_md_equal
+import pytest
+
+TEST_DATA_DIR = Path(__file__).parent
+SPECTESTS_PATH = next(TEST_DATA_DIR.glob("gfm_tables_spec.commit-*.json"))
+SPECTESTS_CASES = tuple(
+    {"name": str(entry["example"]), "md": entry["markdown"]}
+    for entry in json.loads(SPECTESTS_PATH.read_text(encoding="utf-8"))
+)
+
+
+@pytest.mark.parametrize(
+    "entry", SPECTESTS_CASES, ids=[c["name"] for c in SPECTESTS_CASES]
+)
+def test_spec(entry):
+    """Test mdformat-tables against the GFM spec.
+
+    Test that:
+    1. Markdown AST is the same before and after 1 pass of formatting
+    2. Markdown after 1st pass and 2nd pass of formatting are equal
+    """
+    if entry["name"] == "200":
+        pytest.xfail("Need to fix this in markdown-it-py!")
```

Comment:
I assume this is fixed in the develop branch

Reply:
:+1: correct